### PR TITLE
add cp/ edge to supported libs.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:185]: Is this a "soon to be released" repository?

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:174]: Yes

File: packages/tasks/src/model-libraries-snippets.ts
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:184]: I think it is good to leave it as-is for now. Will adapt as they release more.

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:182]: No clue for now, waiting for more clarity.

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:185]: Yes.

File: packages/tasks/src/model-libraries.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:131]:
```suggestion
	},
```

File: packages/tasks/src/model-libraries.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries.ts:187]:
```suggestion
	},
```

**Review by pcuenca — APPROVED** [review]
lgtm with the comments discussed

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries.ts:188]: I'm unsure if this is the correct `library_name` - I will wait for confirmation before the merge.

File: packages/tasks/src/model-libraries-snippets.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts:184]:
```suggestion
outputs = model.generate(inputs.input_ids, max_length=50, top_k=100, top_p=0.99)
```

File: packages/tasks/src/model-libraries-snippets.ts
Comment (pcuenca) [inline packages/tasks/src/model-libraries-snippets.ts:196]: is this necessary? (just curious)

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:196]: According to their official snippet, yes! but I haven't been able to test it till now.

**Comment by Vaibhavs10** [comment]
Merging for now, will make follow-up updates as it comes!

**Pull request merged**
